packages/@okta/vuepress-site/guides/sign-into-spa/configure-the-sdk/index.md

```diff
@@ -5,17 +5,19 @@ title: Configure the SDK
 You'll need two values from the Okta Application and the Developer Console you worked with in <GuideLink link="../create-okta-application">Create an Okta Application</GuideLink>:
 
 * **Client ID** - Find it in the applications list or on the application's **General** tab.
-* **Okta domain** - Find it on the Developer Console dashboard in the upper-right corner. 
+* **Okta domain** - Find it on the Developer Console dashboard in the upper-right corner (shown as `Org URL:`).
```

Comment:
**Okta domain** - Find it on the Developer Console dashboard in the upper-right corner as the **Org URL**.